client/client.go

```diff
@@ -66,27 +66,33 @@ func (c Client) withProject(project string) *Client {
 	return &c
 }
 
-func (c *Client) configureEnabledServices() error {
+func (c *Client) configureEnabledServices() diag.Diagnostics {
 	var esLock sync.Mutex
+	var diags diag.Diagnostics
 	g, ctx := errgroup.WithContext(context.Background())
 	maxGoroutines := limit.GetMaxGoRoutines()
 	goroutinesSem := semaphore.NewWeighted(helpers.Uint64ToInt64(maxGoroutines))
 	for _, p := range c.projects {
 		project := p
 		if err := goroutinesSem.Acquire(ctx, 1); err != nil {
-			return diag.WrapError(err)
+			return diags.Add(diag.WrapError(err))
 		}
 		g.Go(func() error {
 			defer goroutinesSem.Release(1)
 			cl := c.withProject(project)
 			svc, err := cl.fetchEnabledServices(ctx)
 			esLock.Lock()
-			c.EnabledServices[project] = svc
+			if err != nil {
```

Comment:
Do we have to classify this error? For example if it is a diaper error do we really want to continue execution vs if it is a permissions error we definitely want to continue

Reply:
Good point. but I would continue execution in any case in that stage even if it is a dialer error so I can see how more fine-tuned classification can be more bullet proof and we wont miss this in sentry.